### ISSUE TITLE
Monster Spawner triggers blockEventHook

### ISSUE THREAD

**Issue by duggum** [zhuowei/MCPELauncher#349]
Not sure if this is a new bug or a previously unknown side effect, but when a monster spawner spawns a mob, it triggers the blockEventHook with the same parameters as a chest being closed (event type: 1, data: 0).

**Comment by zhuowei** [comment]
Yeah... I need to have individual hooks for those and get rid of block/level event hooks later.
